supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesApiQueryManager.scala

```diff
@@ -1,65 +1,86 @@
+import com.google.api.client.http.HttpHeaders
+import com.google.api.services.genomics.Genomics
+import com.google.api.services.genomics.model.RunPipelineRequest
 import cromwell.backend.impl.jes.Run
 import cromwell.backend.impl.jes.statuspolling.JesApiQueryManager._
+import cromwell.core.CromwellFatalExceptionMarker
 import cromwell.core.Dispatcher.BackendDispatcher
+import cromwell.core.retry.{Backoff, SimpleExponentialBackoff}
+import cromwell.util.StopAndLogSupervisor
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.numeric._
 
+import scala.concurrent.duration._
 import scala.collection.immutable.Queue
 
 /**
-  * Currently, just holds a set of JES status poll requests until a PollingActor pulls the work.
-  * TODO: Could eventually move all of the JES queries into a single work-pulling model.
+  * Holds a set of JES API requests until a JesQueryActor pulls the work.
   */
-class JesApiQueryManager(val qps: Int) extends Actor with ActorLogging {
+class JesApiQueryManager(val qps: Int Refined Positive) extends Actor with ActorLogging with StopAndLogSupervisor {
+
+  private implicit val ec = context.dispatcher
+  private val maxRetries = 10
 
   // workQueue is protected for the unit tests, not intended to be generally overridden
-  protected[statuspolling] var workQueue: Queue[JesStatusPollQuery] = Queue.empty
+  protected[statuspolling] var workQueue: Queue[JesApiQuery] = Queue.empty
   private var workInProgress: Map[ActorRef, JesPollingWorkBatch] = Map.empty
 
-  // If the statusPoller dies, we want to stop it and handle the termination ourselves.
-  override val supervisorStrategy = SupervisorStrategy.stoppingStrategy
   private def statusPollerProps = JesPollingActor.props(self, qps)
 
   // statusPoller is protected for the unit tests, not intended to be generally overridden
   protected[statuspolling] var statusPoller: ActorRef = _
 
-  resetStatusPoller()
+  resetWorker()
 
   override def receive = {
     case DoPoll(run) => workQueue :+= JesStatusPollQuery(sender, run)
+    case DoCreateRun(genomics, rpr) =>
+      workQueue :+= JesRunCreationQuery(sender, genomics, rpr)
+    case q: JesApiQuery => workQueue :+= q
     case RequestJesPollingWork(maxBatchSize) =>
-      log.debug(s"Request for JES Polling Work received (max batch: $maxBatchSize, current queue size is ${workQueue.size})")
+      log.debug("Request for JES Polling Work received (max batch: {}, current queue size is {})", maxBatchSize, workQueue.size)
       handleJesPollingRequest(sender, maxBatchSize)
+    case failure: JesApiQueryFailed => handleQueryFailure(failure)
     case Terminated(actorRef) => handleTerminated(actorRef)
     case other => log.error(s"Unexpected message to JesPollingManager: $other")
   }
 
-  /**
-    * !! Function Impurity Warning: Modifies Actor Data !!
-    */
+  private def handleQueryFailure(failure: JesApiQueryFailed) = failure match {
+    case JesApiQueryFailed(request, _) if request.failedAttempts < maxRetries =>
```

Comment:
TOL: all failures, as long as they are under the `maxRetries` count, are considered retryable ? 
To be clear I don't think it's a big deal because even if someone messes up their conf and get `Forbidden` responses nothing will start and they will have to stop cromwell anyway but just as a note.